test/test_tree_2d_lbm.jl

```diff
@@ -109,25 +94,11 @@ end
                             v2 = 0
 
                             return equilibrium_distribution(rho, v1, v2, equations)
-                            # Ensure that we do not have excessive memory allocations
-                            # (e.g., from type instabilities)
-                            let
-                                t = sol.t[end]
-                                u_ode = sol.u[end]
-                                du_ode = similar(u_ode)
-                                @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) <
-                                      1000
-                            end
```

Comment:
Here was something funky, which probably wasn't supposed to be here. I just deleted it.